install.sh

```diff
@@ -51,6 +50,42 @@ echo_out() { echo -e "\e[0;37m${*}${RESET}" >&1; } # Use Gray for program output
 # Skip all checks if running on a docker container.
 [[ -f "/.dockerenv" ]] && CREW_FORCE_INSTALL=1
 
+if [[ "$CREW_FORCE_INSTALL" ]]; then
```

Comment:
Would it not make sense to gate all of the `CREW_FORCE_INSTALL` checks behind one big `if` statement instead of having multiple?

Reply:
The other conditions ask if `CREW_FORCE_INSTALL` is set manually, so I have left those alone.

But I have combined the parts that make sense combined.